src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -319,3 +319,13 @@ public net.minecraft.client.data.models.ModelProvider$ItemInfoCollector register
 
 # Required to support sorting the client reload listeners.
 private-f net.minecraft.server.packs.resources.ReloadableResourceManager listeners
+
+ # Made public for datagenning these loot functions
+ public net.minecraft.world.level.storage.loot.functions.LootItemConditionalFunction simpleBuilder(Ljava/util/function/Function;)Lnet/minecraft/world/level/storage/loot/functions/LootItemConditionalFunction$Builder; # simpleBuilder
+ public net.minecraft.world.level.storage.loot.functions.FilteredFunction <init>(Ljava/util/List;Lnet/minecraft/advancements/critereon/ItemPredicate;Lnet/minecraft/world/level/storage/loot/functions/LootItemFunction;)V # <init>
+ public net.minecraft.world.level.storage.loot.functions.ModifyContainerContents <init>(Ljava/util/List;Lnet/minecraft/world/level/storage/loot/ContainerComponentManipulator;Lnet/minecraft/world/level/storage/loot/functions/LootItemFunction;)V # <init>
+ public net.minecraft.world.level.storage.loot.functions.SetFireworksFunction <init>(Ljava/util/List;Ljava/util/Optional;Ljava/util/Optional;)V # <init>
+ public net.minecraft.world.level.storage.loot.functions.SetItemFunction <init>(Ljava/util/List;Lnet/minecraft/core/Holder;)V # <init>
+ public net.minecraft.world.level.storage.loot.functions.SetWritableBookPagesFunction <init>(Ljava/util/List;Ljava/util/List;Lnet/minecraft/world/level/storage/loot/functions/ListOperation;)V # <init>
+ public net.minecraft.world.level.storage.loot.functions.SetWrittenBookPagesFunction <init>(Ljava/util/List;Ljava/util/List;Lnet/minecraft/world/level/storage/loot/functions/ListOperation;)V # <init>
+ public net.minecraft.world.level.storage.loot.functions.ToggleTooltips <init>(Ljava/util/List;Ljava/util/Map;)V # <init>
```

Comment:
There is an extra leading space on all these lines; please remove them.

Reply:
Fixed, idk how that happened lol